book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = rentals
+  end
+
+  def add_rentals(person, date)
```

Comment:
- [ ] [OPTIONAL] in terms of naming this function should not be pluralized. the method name should always define what the method does. this method is only adding *one* rental.
```suggestion
  def add_rental(person, date)
```